db/migrate/20200128122000_add_type_column_to_service_order.rb

```diff
@@ -0,0 +1,23 @@
+class AddTypeColumnToServiceOrder < ActiveRecord::Migration[5.0]
+  class ServiceOrder < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+  class MiqRequest < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :service_orders, :type, :string
+
+    say_with_time("Set service_orders type") do
+      v2v_order_ids = MiqRequest.where(:type => 'ServiceTemplateTransformationPlanRequest').pluck(:service_order_id)
```

Comment:
These should all be scoped with `in_my_region`.

Reply:
Done